### PR TITLE
ctl: support to debug hot bucket info

### LINKED ISSUE #6765: pd-ctl: support to debug hot bucket info

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?

Issue Number: Close #6765

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test

```
{
  "1": [
    {
      "start_key": "61",
      "end_key": "62",
      "hot_degree": 1,
      "read_bytes": 1048576,
      "read_keys": 1153433,
      "write_bytes": 1363148,
      "write_keys": 1258291
    }
  ]
}
```

Code changes

Side effects

Related changes

### Release note

```release-note
debug hot bucket info
```

### CODE REVIEW

File: tests/pdctl/helper.go [hunk 1]
```diff
@@ -130,3 +130,24 @@ func MustPutRegion(re *require.Assertions, cluster *tests.TestCluster, regionID,
 	re.NoError(err)
 	return r
 }
+
+// MustReportBuckets is used for test purpose.
+func MustReportBuckets(re *require.Assertions, cluster *tests.TestCluster, regionID uint64, start, end []byte, loads []uint64) *metapb.Buckets {
```
Comment (HuSharp) [inline tests/pdctl/helper.go:135]: I think it's better to pass `BucketStats`
```suggestion
func MustReportBuckets(re *require.Assertions, cluster *tests.TestCluster, regionID uint64, start, end []byte, stats *metapb.BucketStats) *metapb.Buckets {
```

**Review by HuSharp — APPROVED** [review]
LGTM

File: server/handler.go [hunk 2]
```diff
@@ -186,6 +187,16 @@ func (h *Handler) GetHotWriteRegions() *statistics.StoreHotPeersInfos {
 	return c.GetHotWriteRegions()
 }
 
+// GetHotBuckets returns all hot buckets stats.
+func (h *Handler) GetHotBuckets(regionIds ...uint64) map[uint64][]*buckets.BucketStat {
```
Comment (lhy1024) [inline server/handler.go:191]:
```suggestion
func (h *Handler) GetHotBuckets(regionIDs...uint64) map[uint64][]*buckets.BucketStat {
```

**Review by lhy1024 — APPROVED** [review]
the rest LGTM

**Pull request merged**